### ISSUE TITLE
U2F Broken - "could not be translated"

### ISSUE THREAD

**Issue by pavlenex** [btcpayserver/btcpayserver#1269]
**Describe the problem/bug**
After updating to 1.0.3.147 I'm unable to log in via U2F previously set up.

**Your environment** 
* Version of BTCPay Server: 1.0.3.147:
* Deployment method: Docker

**Logs (if applicable)**
```
fail: Microsoft.AspNetCore.Antiforgery.DefaultAntiforgery: An exception was thrown while deserializing the token.
Microsoft.AspNetCore.Antiforgery.AntiforgeryValidationException: The antiforgery token could not be decrypted.
 ---> System.Security.Cryptography.CryptographicException: The key {e0e020cb-6c49-4497-8b37-36c0d972b5f4} was not found in the key ring.
   at Microsoft.AspNetCore.DataProtection.KeyManagement.KeyRingBasedDataProtector.UnprotectCore(Byte[] protectedData, Boolean allowOperationsOnRevokedKeys, UnprotectStatus& status)
   at Microsoft.AspNetCore.DataProtection.KeyManagement.KeyRingBasedDataProtector.DangerousUnprotect(Byte[] protectedData, Boolean ignoreRevocationErrors, Boolean& requiresMigration, Boolean& wasRevoked)
   at Microsoft.AspNetCore.DataProtection.KeyManagement.KeyRingBasedDataProtector.Unprotect(Byte[] protectedData)
   at Microsoft.AspNetCore.Antiforgery.DefaultAntiforgeryTokenSerializer.Deserialize(String serializedToken)
   --- End of inner exception stack trace ---
   at Microsoft.AspNetCore.Antiforgery.DefaultAntiforgeryTokenSerializer.Deserialize(String serializedToken)
   at Microsoft.AspNetCore.Antiforgery.DefaultAntiforgery.GetCookieTokenDoesNotThrow(HttpContext httpContext)
```
```
 The LINQ expression 'DbSet<U2FDevice> .Where(u => u.ApplicationUserId.Equals( value: __8__locals1_userId_0, comparisonType: InvariantCulture) && u.KeyHandle .SequenceEqual(__keyHandle_1))' could not be translated. Either rewrite the query in a form that can be translated, or switch to client evaluation explicitly by inserting a call to either AsEnumerable(), AsAsyncEnumerable(), ToList(), or ToListAsync(). See https://go.microsoft.com/fwlink/?linkid=2101038 for more information.
```
**To Reproduce**
1. Add U2F
2. Update
3. Try to log in

**Screenshots/Links**
![Screenshot 2020-01-13 at 18 42 48](https://user-images.githubusercontent.com/36959754/72278773-04ce0a80-3635-11ea-8e52-a8e63265ef65.png)

**Comment by pavlenex** [comment]
I can login via 2FA btw, but when I try U2F it does not work.

Adding new U2F also does not work and provides the same error.

**Comment by NicolasDorier** [comment]
Fixed in 1.0.3.148 sorry for this!
